pkg/operator/controller/ingress/controller.go

```diff
@@ -674,7 +674,7 @@ func (r *reconciler) ensureIngressDeleted(ingress *operatorv1.IngressController)
 	}
 
 	// Delete the metrics related to the ingresscontroller
-	DeleteIngressControllerConditionsMetric(ingress)
+	DeleteIngressControllerMetrics(ingress)
```

Comment:
I would keep two separate functions, one for controller conditions and one for this new metric.

Reply:
Now you should revert the name change too.